docs/contract-test-collection.md

```diff
@@ -49,7 +49,7 @@ Each contract description will include the storage in Michelson and the storage
 
 ## IncrementContract
 
-[KT1NcdpzokZQY4sLmCBUwLnMHQCCQ6rRXYwS](https://better-call.dev/hangzhou2net/KT1NcdpzokZQY4sLmCBUwLnMHQCCQ6rRXYwS/code)
```

Comment:
Should the contract addresses be updated? They all have the same address as for hangzhounet.

Reply:
I have updated them.